lms/static/js/spec/student_account/account_settings_spec.js

```diff
@@ -0,0 +1,42 @@
+define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpers/template_helpers',
```

Comment:
I can't see any test in this file or at-least any expectation. Are we suppose to remove this file or we need to add tests/expectations?